CI/tests/test_cpu_hog_gh.sh

```diff
@@ -0,0 +1,20 @@
+set -xeEo pipefail
```

Comment:
Do you think we can remove the hog scenario scripts given that they are disabled for the runs? Thoughts?

Reply:
do you want me to remove all the hog scenarios or only the _gh that I added? If that's the case should I remove all the tests that are not ran? I was keeping them just in case we figure out how to make them work.

Reply:
Let's add for them for now @tsebastiani. Thanks.